docs/kb/semgrep-ci/azure-using-templates-with-semgrep.md

```diff
@@ -0,0 +1,75 @@
+---
+tags:
+  - Azure
```

Comment:
```suggestion
  - Azure Pipelines
```
Just suggesting we specify the product because we also have to differentiate Azure Active Directory and Azure Repos 👍